client/src/components/FeaturesDetails.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class FeaturesDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      focusedFeature: 0,
+    };
+  }
+
+  render() {
+    return (
+      <div className="FeaturesDetails">
+        {this.props.features.map((feature, index) => <div key={index}>{feature.value}</div>)}
+      </div>
+    );
+  }
+}
```

Comment:
So related to the upper comment, I think you have to set propTypes and defaultProps in this component.